pkg/vpcmodel/externalNetwork.go

```diff
@@ -87,10 +87,10 @@ func getPublicInternetIPblocksList() (internetIPblocksList []*common.IPBlock, al
 	return ipStringsToIPblocks(publicInternetAddressList)
 }
 
-func newExternalNode(isPublicInternet bool, ipb *common.IPBlock) (Node, error) {
+func NewExternalNode(isPublicInternet bool, ipb *common.IPBlock) (Node, error) {
```

Comment:
why does it need to be exported if it's used in the same pkg?

Reply:
In some point I used it, and then did not need to use it and forget to un export it. Will fix.